lib/ecom/api.ts

```diff
@@ -243,17 +249,10 @@ function createEcomApi(wixClient: WixApiClient): EcomAPI {
         },
         async getOrder(id) {
             try {
-                const order = await wixClient.orders.getOrder(id);
-                if (!order) {
-                    return failureResponse(EcomApiErrorCodes.OrderNotFound, 'Order not found');
-                }
-
-                return successResponse(order);
-            } catch (e) {
-                if (isEcomSDKError(e) && e.details.applicationError.code === 404) {
```

Comment:
> `if (isEcomSDKError(e) && e.details.applicationError.code === 404)`

Note: this condition was incorrect, `WixClient` returns `"NOT_FOUND"` instead of `404` for this method 🤦‍♂️